src/Ninja/G1CP/Content/Tests/test149.d

```diff
@@ -0,0 +1,38 @@
+/*
+ * #149 Spelling - Improved Ore Armor (DE/EN)
+ *
+ * The text of the item "ORE_ARMOR_H" is inspected programmatically.
+ *
+ * Expected behavior: The armor will have the correct description (checked for English and German localization only).
```

Comment:
Here it says "description" but it is the name in this fix. Maybe we should make these comments more generic. I think it is very tedious to adjust these comments every time (or potentially have to go back and fix them if we forgot). We can keep that in mind going forward.